locations/magda.yml

```diff
@@ -102,35 +122,52 @@ networks:
       magda-ap-remise: 9
       magda-switch-unten: 10
 
-  - vid: 40
+  - vid: 43
     role: dhcp
-    prefix: 10.31.83.192/26
-    ipv6_subprefix: 0
+    name: remise
+    prefix: 10.31.214.128/28
+    ipv6_subprefix: 41
     inbound_filtering: true
-    enforce_client_isolation: true
+    enforce_client_isolation: false
     assignments:
       magda-core: 1
 
-  - vid: 41
-    role: dhcp
-    name: pubdhcp
-    prefix: 10.248.42.48/28
-    ipv6_subprefix: 2
-    inbound_filtering: false
-    enforce_client_isolation: true
-    assignments:
-      magda-core: 1
 
 location__channel_assignments_11a_standard__to_merge:
   magda-ap1: 48-20
   magda-ap2: 36-20
   magda-ap3: 44-20
   magda-ap-remise: 36-40
 
-# channel-bandwith-txpower in dbm
+# channel-bandwidth-txpower in dbm
 
 # ap1: hof - 5ghz
 # ap2: knast - 5ghz
 # ap3: kirche - 5ghz
 # ap4: knast - 2ghz
 # ap-remise: remise via haus netzwerk via media converter - dual band
+
+location__wireless_profiles__to_merge:
+  - name: remise
+    ifaces:
+      - mode: ap
+        ssid: berlin.freifunk.net
+        encryption: none
+        network: dhcp
+        radio: [11a_standard, 11g_standard]
+        ifname_hint: ff
+
+      - mode: ap
+        ssid: berlin.freifunk.net Encrypted
+        encryption: owe
+        network: dhcp
+        radio: [11a_standard, 11g_standard]
+        ifname_hint: ffowe
+        ieee80211w: 2
+
+      - mode: ap
+        ssid: Remise
+        encryption: none
+        network: prdhcp
```

Comment:
Wrong `network` and `ifname_hint`  (should be `remise` I guess)

LGTM otherwise :+1: